modules/administration/pages/content-lifecycle.adoc

```diff
@@ -120,6 +120,16 @@ When you have created your project, defined environments, and attached sources a
 Building applies filters to the attached sources and clones them to the first environment in the project.
 
 
+[NOTE]
+====
+About patch sharing
+
+When you clone a vendor channel multiple times in your organization (e.g. you have 2 content projects with the same source vendor channels), {productname} does not create new patch clones for each cloned channel. Instead of that, a single patch clone is shared between your cloned channels.
+
+One important consequence is, that if the vendor modifies a patch (e.g. the patch gets retracted, or the patch packages change) and you build one content project, all channels sharing the cloned patch be synchronized with the original (even the channels in other environments of your content project, or other content project channels in your organization).
+
+You can turn the automatic patch synchronization feature off in your organization settings. You can synchronize the patch later manually using the Patch Sync function (menu:Software[Manage > Channels], click the channel you want to synchronize, and follow the menu:Repositories[Sync]).
```

Comment:
But synchronize a patch will still affect all channels where this patch exists in. Only the point in time when this happens is different.

Reply:
Ok, maybe it deserves a more detailed explanation. Thanks for the comment!